lib/core/wrappers.sh

```diff
@@ -1,9 +1,9 @@
 
 
 function create_wrappers() {
-    mkdir -p ${JUNEST_HOME}/usr/bin_wrappers
+    mkdir -p "${JUNEST_HOME}/usr/bin_wrappers"
 
-    cd ${JUNEST_HOME}/usr/bin
+    cd "${JUNEST_HOME}/usr/bin" || return 1
```

Comment:
No need to use `return 1`. The script has `set -e` to fail fast when any error occurs.

Reply:
It is seems that the rationale is only related to get a warning message when using shellcheck https://github.com/koalaman/shellcheck/issues/466 . I think `set -e` should do the trick plus relying on placing always `exit`/`return` is just tedious and hard to enforce. Approving anyway given it is a minor thing.